src/pages/VenuePage/VenuePage.tsx

```diff
@@ -295,7 +295,7 @@ const VenuePage = () => {
     return <LoadingPage />;
   }
 
-  if (!(profile?.partyName && profile?.pictureUrl)) {
+  if (profile && !(profile?.partyName && profile?.pictureUrl)) {
     history.push(`/account/profile?venueId=${venueId}`);
```

Comment:
I had review comments on making this more robust/self-documenting, but seems @thecadams has done it already in https://github.com/sparkletown/sparkle/pull/934#discussion_r538775630